builds.yml

```diff
@@ -288,3 +288,8 @@ env:
   ###
 
   - EIP_4337_ENTRYPOINT: null
+  ###
+  # Enable/disable why did you render debug tool: https://github.com/welldone-software/why-did-you-render
+  # This should NEVER be enabled in production since it slows down react
+  ###
+  - WHY_DID_YOU_RENDER_ENABLED: false
```

Comment:
Precedent (ENABLE_NETWORK_UI_REDESIGN,ENABLE_CONFIRMATION_REDESIGN, ENABLE_SETTINGS_PAGE_DEV_OPTIONS) makes me think this should be named "ENABLE_WHY_DID_YOU_RENDER".

Reply:
fair point